apps/theming-designer/src/components/AccChecker.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+//import { ColorsPage } from '@uifabric/fabric-website-resources/lib/components/pages/ColorsPage';
+
+export class AccChecker extends React.Component {
```

Comment:
Prefer fully qualified names to help with readability for developers less familiar with the functionality.